pytse_client/orderbook/common.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import logging
+import os
+import pandas as pd
+from pathlib import Path
+from pytse_client.ticker import Ticker
+from pytse_client.config import ORDER_BOOK_HIST_PATH
+
+# maximum available depth in tsetmc for historical orderbook
+MAX_DEPTH = 5
+ERROR_MSG = "{date} is not a valid date object. Make sure it was a trade day."
+
+logging.basicConfig(
```

Comment:
This is a very small point. For logging it's better to use a named logger instance in the libraries to minimize the side effects in the code that is using the library. You can see how we are doing this in other files, it's basically getting a logger with a name.

https://docs.python.org/3/howto/logging-cookbook.html#customizing-logrecord

![Uploading image.png…]()

Reply:
Great point. I will fix this.